rotkehlchen/tests/unit/decoders/test_optimism_bridge.py

```diff
@@ -405,3 +408,155 @@ def test_prove_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+            notes=f'Burned {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:10/erc20:0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1'),
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=user_address,
+            notes=f'Bridge {deposit_amount} DAI from Optimism to Ethereum via Optimism bridge',
+            counterparty=CPT_OPTIMISM,
+            address=ZERO_ADDRESS,
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_withdraw_dai_on_ethereum(database, ethereum_inquirer, ethereum_accounts):
+    """DAI uses special bridge. See https://github.com/makerdao/optimism-dai-bridge"""
+    evmhash = deserialize_evm_tx_hash('0x3e958996b7c4b2466ca9521d4c688ff4817c28d66bb3f52ec1174c6d0f1c44db')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address, timestamp, gas_amount, recieve_amount = ethereum_accounts[0], TimestampMS(1671053711000), '0.0097257501', '106317.949168317256453804'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burned {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=876,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:1/erc20:0x6B175474E89094C44Da98b954EedeAC495271d0F'),
+            balance=Balance(amount=FVal(recieve_amount)),
+            location_label=user_address,
+            notes=f'Bridge {recieve_amount} DAI from Optimism to Ethereum via Optimism bridge',
+            counterparty=CPT_OPTIMISM,
+            address=OPTIMISM_L1_ESKROW,
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('optimism_accounts', [['0x301605C95acbED7A1fD9C2c0DeEe964e2AFBd0C3']])
+def test_withdraw_dai_on_optimism(database, optimism_inquirer, optimism_accounts):
+    """DAI uses special bridge. See https://github.com/makerdao/optimism-dai-bridge"""
```

Comment:
no need to copy paste the same docstring in all the tests. I would add it to the docstring of `_decode_bridge` where it is more useful and remove it from the tests